index.html

```diff
@@ -219,6 +219,119 @@ <h3>Skills</h3>
 				</div>
 			</div>
 		</section>
+		<footer id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label>
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
+			</form>
+			<form
+				action="https://formspree.io/f/xpznwejk"
+				method="post"
+				class="form2"
+			>
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstN"
+							id="firstN"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastN"
+							id="lastN"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
+			</form>
```

Comment:
- It is not advisable to have two forms on your page, I suggest having just one `form` on your HTML page.Please fix this pray 👍 👍

Reply:
Hello @mckent05, Thanks for your review, I have tried a couple of times having a single form in the page. The form has a single name input in mobile version and two name input in desktop version so when i trying hiding the inputs using "display: none"  the form doesn't submit because of the validation, My assumption is the form doesn't submit because the hidden element are not validated when submitting the form...am open to suggestions but i don't think i will be modifying the form since i have tried different methods to ensure that i have a single form but none is working